suite-native/atoms/src/Pictogram/Pictogram.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+import { Icon, IconName } from '@suite-native/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Color } from '@trezor/theme';
+
+import { Box } from '../Box';
+import { CriticalIconSvg } from './CriticalIconSvg';
+import { CriticalShapeSvg } from './CriticalShapeSvg';
+import { InfoIconSvg } from './InfoIconSvg';
+import { InfoShapeSvg } from './InfoShapeSvg';
+import { SuccessIconSvg } from './SuccessIconSvg';
+import { SuccessShapeSvg } from './SuccessShapeSvg';
+import { WarningIconSvg } from './WarningIconSvg';
+import { WarningShapeSvg } from './WarningShapeSvg';
+
+export type PictogramVariant = 'success' | 'info' | 'warning' | 'critical';
+
+type PictogramProps = {
+    variant: PictogramVariant;
+    icon?: IconName;
+};
+
+type PictogramConfig = {
+    ShapeSvg:
```

Comment:
these unions have no benefit: 

![Snímek obrazovky 2024-12-09 v 11 48 28](https://github.com/user-attachments/assets/e36a5754-3750-4146-9ba5-07cdbf4bfa53)

Reply:
I thought about before but didn't want to deal with it... Here you go: https://github.com/trezor/trezor-suite/pull/15832/commits/9af8856c4fdd65d7b6439de0820d5ddc1c428d2e